spec/helpers/sticky_footer_visibility_spec.rb

```diff
@@ -0,0 +1,24 @@
+RSpec.describe StickyNewsletterVisibility, type: :helper do
+  let(:request) { double('request')}
+
+  describe 'home page' do
+    %w(en cy).each do |locale|
+      it 'excludes in any language' do
+        allow(request).to receive(:url) { "http://example.com/#{locale}" }
+        allow(request).to receive(:base_url) { 'http://example.com' }
+        expect(display_sticky_newsletter_on_this_slug?).to be false
+      end
+    end
+  end
+
+  describe 'category slugs' do
+    %w(en cy).each do |locale|
+      it 'excludes in any language' do
+        allow(request).to receive(:url) { "http://example.com/#{locale}/categories/insurance" }
+        allow(request).to receive(:base_url) { 'http://example.com' }
+
+        expect(display_sticky_newsletter_on_this_slug?).to be false
+      end
+    end
+  end
+end
```

Comment:
Hey @tomas-stefano yay! It works. Checkout the changes here: https://github.com/moneyadviceservice/frontend/commit/7b6d6d5dc5aaa05536624aa3b67f552cf9bb4e80